src/lib/BaseCalc.ts

```diff
@@ -527,6 +527,14 @@ export default class BaseCalc {
     ]);
   }
 
+  /**
+   * Whether the player is using an eclipse atlatl.
+   * @see https://oldschool.runescape.wiki/w/Eclipse_atlatl
+   */
+  protected isWearingAtlatl(): boolean {
+    return this.wearing('Eclipse atlatl');
```

Comment:
Since there's only one entry here, just inline it wherever it's used instead of making a new fn